### DISCUSSION TITLE
Freetube uses QUIC and/or Kyber protocols?

### DISCUSSION THREAD

**Discussion by AndyShade** [FreeTubeApp/FreeTube#5513] [Q&A]
and if uses, can they disabled without recompile app a- how?
If not, I'd be grateful for a patch that does that.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Yes we do use HTTP/3 protocol and no we dont use Kyber protocol.

>  I'd be grateful for a patch that does that.

duplicate of https://github.com/FreeTubeApp/FreeTube/issues/2336
